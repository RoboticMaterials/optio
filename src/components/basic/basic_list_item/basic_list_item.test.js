import React from 'react';
import BasicListItem from './basic_list_item';
import {shallow, mount} from 'enzyme'

it('renders without crashing', () => {
  shallow(<BasicListItem/>)
})

it('renders correctly', () => {
  const tree = shallow(<BasicListItem/>)
  expect(JSON.stringify(tree)).toMatchSnapshot()
})
