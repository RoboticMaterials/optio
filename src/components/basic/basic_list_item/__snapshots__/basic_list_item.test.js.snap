// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches the stored UI snapshot 1`] = `
<div
  className="sc-bdnxRM buvdtd"
>
  <div
    className="sc-hKFxyN hjidFM"
  />
  <div
    className="sc-eCApnc kikrXR content-container"
  >
    <span
      className="sc-gtsrHT gaJkdo title"
    >
      
    </span>
    <span
      className="sc-dlnjwi fcRUjP status"
    />
  </div>
  <div
    className="sc-jSFjdj hdjpDT"
  />
</div>
`;
